public/main.js

```diff
@@ -158,162 +158,165 @@ Contact.prototype.onShowContactlistClicked = function (event) {
 	this.onCancelClicked();
 	
 	var editInput = 
-		"<form class='edit-item-form' hidden>" +
+	  "<form class='edit-item-form' hidden>" +
 	          "<input type='text' name='edit-item-input'" +
 	               "title='Edit item'>" + 
 	          "<button type='submit' class='visually-hidden'>Edit item</button>" +
 	    "</form>";
-
+	
 	this.showUl.html("");
 	var listItem = $(event.target).parent();
 	var i = this.listContact.find("li").index(listItem);
 	this.showUlid = i;
 	
 	this.showUl.prepend(
-		"<h4>Double-click to edit information</h4>" +
-		"<h2>" + this.persons[i].firstName + " " + this.persons[i].lastName + "</h2>"
+	  "<h4>Double-click to edit information</h4>" +
+	  "<h2>" + this.persons[i].firstName + " " + this.persons[i].lastName + "</h2>"
 	);
 	var showStr = "<ul>First name: <li><div class='value'>" + this.persons[i].firstName + "</div>" + editInput + "</li></ul>" +
-		"<ul>Last name: <li><div class='value'>" + this.persons[i].lastName + "</div>" + editInput + "</li></ul>" +
-		"<ul>Phone Number:"; 
+	  "<ul>Last name: <li><div class='value'>" + this.persons[i].lastName + "</div>" + editInput + "</li></ul>" +
+	  "<ul>Phone Number:"; 
 	for(var j = 0; j < this.persons[i].phoneNumber.length; j += 1)
-		showStr += "<li><div class = 'value'>" + this.persons[i].phoneNumber[j] + "</div>" + editInput + "</li>";
+	  showStr += "<li><div class = 'value'>" + this.persons[i].phoneNumber[j] + "</div>" + editInput + "</li>";
 	showStr += "</ul><ul>Address:";
 	if(this.persons[i]["Address"][0]) {
-		for(var j = 0; j < this.persons[i]["Address"].length; j += 1)
-			showStr += "<li><div class='value'>" + this.persons[i].Address[j] + "</div>" + editInput + "</li>";
+	  for(var j = 0; j < this.persons[i]["Address"].length; j += 1)
+	    showStr += "<li><div class='value'>" + this.persons[i].Address[j] + "</div>" + editInput + "</li>";
 	}
 	showStr += "</ul>";
 	this.showUl.append(showStr);
 };
 
 Contact.prototype.addItem = function(person) {
-    if(person) {
+	if(person) {
 	    var ajax = $.ajax('/items', {
 	        type: 'POST',
 	        data: JSON.stringify(person),
 	        dataType: 'json',
 	        contentType: 'application/json'
 	    });
 	    ajax.done(this.getItems.bind(this));
-    }
+	}
 };
 
 Contact.prototype.getItems = function() {
-    var ajax = $.ajax('/items', {
-        type: 'GET',
-        dataType: 'json'
-    });
-    ajax.done(this.onGetItemsDone.bind(this));
+	var ajax = $.ajax('/items', {
+	    type: 'GET',
+	    dataType: 'json'
+	});
+	ajax.done(this.onGetItemsDone.bind(this));
```

Comment:
this will _always_ run. You should have a success/error condition here, so the app won't break if the server doesn't respond correctly.